tests/director/tests/application_templates_subscription_test.go

```diff
@@ -125,7 +138,8 @@ func TestSubscriptionApplicationTemplateFlow(baseT *testing.T) {
 
 			// THEN
 			consumerToken := token.GetUserToken(t, ctx, conf.ConsumerTokenURL+conf.TokenPath, conf.ProviderClientID, conf.ProviderClientSecret, conf.BasicUsername, conf.BasicPassword, "subscriptionClaims")
-			headers := map[string][]string{subscription.AuthorizationHeader: {fmt.Sprintf("Bearer %s", consumerToken)}}
+			consumerClaims := token.FlattenTokenClaims(stdT, consumerToken)
+			headers := map[string][]string{subscription.UserContextHeader: {consumerClaims}}
```

Comment:
Currently, all app_template tests are adapted to send the new header instead of the whole token? Shouldn't we test both calls (with header & with token) separately or we decided to adapt and test only the header calls?

Reply:
The applicatio template will be using the new functionality with the `user_context` header and that's why we're testing it only with the header. On the other hand in ord-service tests, we have a "subscription" test with cert + token, and that way we cover both approaches with the e2e tests. But to be more robust/confident in our tests, there is one more test case that uses cert + user_context header to be sure we cover both approaches in "our" tests and we're not relying only on app template tests